src/Mapbender/RoutingBundle/Element/Routing.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Mapbender\RoutingBundle\Element;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Exception;
+use Mapbender\Component\Element\AbstractElementService;
+use Mapbender\Component\Element\TemplateView;
+use Mapbender\CoreBundle\Entity\Element;
+use Mapbender\RoutingBundle\Component\RoutingHandler;
+use Mapbender\RoutingBundle\Component\SearchHandler;
+use Mapbender\RoutingBundle\Component\ReverseGeocodingHandler;
+
+/**
+ * Class RoutingElement
+ * @package Mapbender\RoutingBundle\Element
+ * @author Christian Kuntzsch <christian.kuntzsch@wheregroup.com>
+ * @author Robert Klemm <robert.klemm@wheregroup.com>
+ */
+class Routing extends AbstractElementService
+{
+    protected RoutingHandler $routingHandler;
+
+    protected SearchHandler $searchHandler;
+
+    protected ReverseGeocodingHandler $reverseGeocodingHandler;
+
+    public function __construct(RoutingHandler $routingHandler, SearchHandler $searchHandler, ReverseGeocodingHandler $reverseGeocodingHandler) {
+        $this->routingHandler = $routingHandler;
+        $this->searchHandler = $searchHandler;
+        $this->reverseGeocodingHandler = $reverseGeocodingHandler;
+    }
+
+    /**
+     * @return string
```

Comment:
this comments are superflous when using native return types

Reply:
I removed them.